ENIGMAsystem/SHELL/Universal_System/Resources/fonts_internal.h

```diff
@@ -48,11 +48,15 @@ namespace enigma
   };
   class font
   {
-    bool destroyed = false;
+    bool destroyed;
```

Comment:
No need for this. Or if there is, there shouldn't be; I told Robert how to avoid this.